WordPress/src/main/res/layout/backup_download_details_fragment.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/backup_details_scroll_view"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:fillViewport="true"
+    android:padding="@dimen/margin_medium_large">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/backup_details_container"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <ImageView
+            android:id="@+id/backup_details_action_icon"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:contentDescription="@string/backup_download_details_icon_content_description"
+            app:layout_constraintStart_toStartOf="@+id/backup_details_header"
+            app:layout_constraintTop_toTopOf="parent"
+            tools:src="@drawable/ic_cloud_white_24dp" />
+
+        <com.google.android.material.button.MaterialButton
+            android:id="@+id/backup_details_action_button"
```

Comment:
Minor (🔍 ): Consider moving this view below the `backup_details_description`. This is just so that the code order ends up being consistent with the UI order on screen.